koku/subs/trino_sql/azure_subs_pre_or_clause.sql

```diff
@@ -0,0 +1,36 @@
+SELECT
+  *,
+  COALESCE(date, usagedatetime) as subs_start_time,
+  date_add('day', 1, COALESCE(date, usagedatetime)) as subs_end_time,
+  json_extract_scalar(lower(additionalinfo), '$.vcpus') as subs_vcpu,
+  COALESCE(subscriptionid, subscriptionguid) as subs_account,
```

Comment:
I'm going to assume we want those NULLIF's here too
```suggestion
  COALESCE(NULLIF(subscriptionid, ""), subscriptionguid) as subs_account,
```

Reply:
Added the `NULLIF`